### PR TITLE
Update to apply latest changes

### PR DESCRIPTION
**Pull request by Nashaath**
## Purpose
Update to apply the latest changes.

### CODE REVIEW

File: en/docs/get-started/quick-start-guide.md [hunk 1]
```diff
@@ -1,124 +1,134 @@
 # Quick Start Guide
 
-Choreo is a full lifecycle cloud-native developer platform that enables your developers to create, deploy,  run, and govern APIs, integrations, and microservices on Kubernetes.
+Choreo is a full lifecycle cloud-native developer platform that enables developers to create, deploy, run, and govern APIs, integrations, and microservices on Kubernetes.
 
-This guide walks you through the following:
+In this quick start guide, you will explore how to expose a service endpoint as a REST API via Choreo and securely consume the API from a web application. 
+You will build a simple reading list web application with a sign-in page and functionality to interact with a secure backend REST API. You will use Asgardeo, WSO2's Identity as a Service (IDaaS) solution, to secure user authentication to the web application. The application will allow users to sign in and view the reading list. On signing in, a user can add books to a reading list and delete books from the reading list. The application will also allow users to sign out of the application.
```
Comment (shaniR) [inline en/docs/get-started/quick-start-guide.md:6]: we are repeating signing in here..

File: en/docs/get-started/quick-start-guide.md [hunk 2]
```diff
@@ -1,124 +1,134 @@
...
+    4. Read and accept the privacy policy and terms of use.
 
     5. Click **Create** to add the new organization.
 
-    You will be viewing the **Home** page of the Choreo Console.
+    This takes you to the **Home** page of the Choreo Console.
+
+    !!! info "Enable Asgardeo as the key manager"
+         
+         If you created your organization in Choreo before the 21st of February 2023, and you have not already enabled Asgardeo as the key manager, follow these steps to enable Asgardeo as the default key manager:
```
Comment (shaniR) [inline en/docs/get-started/quick-start-guide.md:42]: remembering dates is difficult, isn't there anyother way for them to check if they have enabled Asgardeo as the default Key Manager?

File: en/docs/get-started/quick-start-guide.md [hunk 3]
```diff
@@ -1,124 +1,134 @@
...
-3. In the **Configure & Deploy** pane, click **Deploy** without entering a sandbox endpoint. 
+3. In the **Configure & Deploy** pane that opens, you can see the **Readinglist** endpoint ready to be deployed. Click the edit icon next to the **Readinglist** endpoint.
+
+4. Change the **Network Visibility** to **Public**. This allows you to expose the endpoint to your web application securely.
+
+5. Click **Update**.
 
     !!! info
-        In this example, you are testing the REST API only in the Choreo Console and not in a sandbox environment. Therefore, you do not need to enter a sandbox endpoint.
+         In this example, you deploy a Ballerina service as a REST endpoint. Therefore, the REST endpoint is generated automatically. If you deploy a non-Ballerina service, you must manually add the REST endpoint and set the network visibility to **Public**.
```
Comment (shaniR) [inline en/docs/get-started/quick-start-guide.md:114]: deployed?

Comment (shaniR) [reply en/docs/get-started/quick-start-guide.md:114]: Shall we point them to the endpoint page, For e.g. learn more about endpoints or something along that lines?

File: en/docs/get-started/quick-start-guide.md [hunk 4]
```diff
@@ -136,262 +146,192 @@ Let's test the `readingList` REST API via Choreo's Open API Console by following
...
-### Step 1.4: Publish the REST API
+### Step 1.4: Publish the service
 
-Now that your API is tested, let's publish it and make it available for applications to consume.
+Now you can publish the tested REST endpoint to make it available for the web application to consume.
 
 #### Step 1.4.1: Update the CORS configuration
 
-The application developer in this scenario calls the API from a different domain to Choreo (i.e., Vercel). By default, web browsers block these calls for security reasons. To enable the application to call the API, update the CORS configuration as follows:
+Since you expect to deploy the web application in Vercel, you would want to call the **Readinglist** backend service deployed in Choreo through the web application on a Vercel domain. By default, web browsers block such calls for security reasons. To enable the web application to call the backend service, you must update the CORS configuration as follows:
```
Comment (shaniR) [inline en/docs/get-started/quick-start-guide.md:162]: "Since you expect to deploy the web application in Vercel" -> We don't set this expectation before hand do we? Shall we rephrase this to imply that you will be doing this, rather than making it sound as it's an expectation of the user?

**Review by shaniR — CHANGES_REQUESTED** [review]

File: en/docs/get-started/quick-start-guide.md (hunk 1 above)
Comment (Nashaath) [reply en/docs/get-started/quick-start-guide.md]: Rephrased the first sentence to remove the sign-in page aspect.

File: en/docs/get-started/quick-start-guide.md (hunk 2 above)
Comment (Nashaath) [reply en/docs/get-started/quick-start-guide.md:42]: AFAIK there is a banner if a user has not enabled Asgardeo as KM. This note is applicable to the very few who have not enabled it even with the banner.

File: en/docs/get-started/quick-start-guide.md (hunk 3 above)
Comment (Nashaath) [reply en/docs/get-started/quick-start-guide.md]: "Deployed" <- - - At this point, the service is not deployed. it will be deployed via the next step. 

+1 to point to the info on endpoints. Added a learn more link.

File: en/docs/get-started/quick-start-guide.md (hunk 4 above)
Comment (Nashaath) [reply en/docs/get-started/quick-start-guide.md]: +1
Rephrase the sentence.

File: en/docs/get-started/quick-start-guide.md (hunk 2 above)
Comment (Nashaath) [reply en/docs/get-started/quick-start-guide.md:42]: @Asitha is there any other way that we can mention to help users to check whether they have enabled Asgardeo as KM?

**Review by shaniR — APPROVED** [review]
